src/App.tsx

```diff
@@ -1,24 +1,208 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState, useEffect, useMemo, useCallback,
+} from 'react';
+import classNames from 'classnames';
+import { Error } from './types/Errors';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { Loader } from './component/Loader/Loader';
+import { TodoList } from './component/TodoList/TodoList';
+import { Footer } from './component/Footer/Footer';
+import { LoadType } from './types/LoadType';
+import { Header } from './component/Header/Header';
 
-const USER_ID = 0;
+const USER_ID = 6908;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [typeOfLoad, setTypeOfLoad] = useState<LoadType>(LoadType.All);
+  const [isLoading, setIsLoading] = useState(true);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [disableInput, setDisableInput] = useState(false);
+  const [loadId, setLoadId] = useState<number[]>([]);
+
+  const getTodosFromServer = async (userId: number) => {
+    try {
+      setIsLoading(true);
+      const getTodo = await getTodos(userId);
+
+      setTodos(getTodo);
+    } catch {
+      setErrorMessage(Error.Load);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const addNewTodo = useCallback(async (title: string) => {
+    if (!title.trim()) {
+      setErrorMessage('Title can not be empty');
```

Comment:
Move error messages to an enum, that way will allow you to avoid magic strings, to be sure that you've not mistaken, and to maintain it in one place.

Reply:
Especially if you already have such enum